.nextmv/update_apps.py

```diff
@@ -130,16 +140,31 @@ def get_manifest(
     return yaml.safe_load(result["Body"].read())
 
 
-def tar_app(name: str, version: str) -> str:
-    """Create a tarbal of the app. Returns the name of the tarball."""
+def push_app(name: str, version: str):
+    """Pushes the app to the Nextmv Marketplace."""
 
-    app_dir = os.path.join(os.getcwd(), "..", name)
-    filename = f"{name}_{version}.tar.gz"
+    workflow_info = app_workflow_info(name)
+    app_id = workflow_info["app_id"]
+    marketplace_app_id = workflow_info["marketplace_app_id"]
+    if app_id == "" or marketplace_app_id == "":
+        return
 
-    with tarfile.open(filename, "w:gz") as tar:
-        tar.add(app_dir, arcname=os.path.basename(app_dir))
+    try:
+        _ = subprocess.run(
+            ["bash", "push_app.sh"],
+            env={
+                "APP_DIR": os.path.join("..", name),
+                "APP_ID": app_id,
+                "MARKETPLACE_APP_ID": marketplace_app_id,
+                "VERSION_ID": version,
+            },
```

Comment:
```suggestion
            env=os.environ | {
                "APP_DIR": os.path.join("..", name),
                "APP_ID": app_id,
                "MARKETPLACE_APP_ID": marketplace_app_id,
                "VERSION_ID": version,
            },
```

Maybe it's already enough, if you forward the original env vars into the subprocess. Alternatively, we could just explicitly set the `PATH` var here to point to `$HOME/.nextmv/`.

Reply:
Merging the two dicts worked great. I should have thought more about passing the env doesn't forward the parent's vars to the subprocess. Late night ignorance I guess.

Reply:
Been there. :100: